### PR TITLE
Fix sequential slugging bug

### PR DESCRIPTION
**Pull request by tekin**
Fixes a bug where a slug conflicting with a slug that starts with a number would get incorrectly sequenced based on the number at the start of the slug, rather than as a fresh sequence. e.g. a conflict with `"2010-2015-slug"` would result in the next sequential slug being `"2010-2015-slug--2011"` instead of the expected `"2010-2015-slug--2"`

### CODE REVIEW

File: test/unit/whitehall/slugging_test.rb [hunk 1]
```diff
@@ -10,9 +10,14 @@ class SluggingTest < ActiveSupport::TestCase
...
-    assert_match /--2/, document2.slug
+    assert_equal 'slug-conflict--2', document2.slug
   end
 
+  test 'should handle document titles with numbers' do
+    document = create(:document, sluggable_string: "2010-2015 conflict")
+    document2 = create(:document, sluggable_string: "2010-2015 conflict")
+    assert_equal '2010-2015-conflict--2', document2.slug
+  end
   test "should strip punctuation properly" do
```
Comment (elliotcm) [inline test/unit/whitehall/slugging_test.rb]: Missing newline.

File: lib/friendly_id/sequential_slug_generator.rb [hunk 2]
```diff
@@ -24,7 +24,11 @@ def next_sequence_number
     end
 
     def last_sequence_number
-      slug_conflicts.last.split("#{slug}#{sequence_separator}").last.to_i
+      if slug_conflicts.size > 1
+        slug_conflicts.last.split("#{slug}#{sequence_separator}").last.to_i
+      else
+        0
+      end
```
Comment (elliotcm) [inline lib/friendly_id/sequential_slug_generator.rb]: Could we regex this instead?  Maybe not, but maybe?

Comment (tekin) [inline lib/friendly_id/sequential_slug_generator.rb]: Probably... I'll have a look.

**Comment by tekin** [comment]
Now with added Rexexp.

**Comment by elliotcm** [comment]
:+1:

**Pull request merged**